README.md

```diff
@@ -1,8 +1,18 @@
 # Hyper Converged Infrastructure Ready Architecture on Red Hat Openstack
-
+	
 
 ## Description
 
+Hyper-convergence is a type of infrastructure system that is largely software-defined with tightly-integrated compute, storage, networking, and virtualization resources. 
+With hyper-convergence in place, customers just don’t eliminate traditional storage infrastructure but they do gain far more efficient management constructs, configurations, monitoring, and troubleshooting. The result is that hyper-convergence is easier and less time-consuming to manage than traditional infrastructure.
+
+Dell EMC Hyper Converged Infrastructure for Red Hat OpenStack Platform is designed in such way that by integrating compute and storage together on a single aggregated 
+cluster, making it a well suited solution for low-footprint remote or central office installation and edge computing. Hyper-Converged Infrastructures (HCI) uses Red Hat 
+Ceph Storage as a storage provider. This infrastructure features hyper-converged nodes, where Compute and Ceph Storage services are collocated and configured for optimized resource usage.
+
+With this repository, user should be able to access kick-start files, templates, validation script and documentation which are needed to perform deployment
```

Comment:
With this repository, users should be able to access kick-start files, templates, validation scripts and documentation which are needed to perform a deployment